<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>600</width>
                <height>400</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Activity Tracker</string>
        </property>
        <widget class="QWidget" name="centralWidget">
            <widget class="QTabWidget" name="tabPages">
                <property name="geometry">
                    <rect>
                        <x>0</x>
                        <y>10</y>
                        <width>591</width>
                        <height>381</height>
                    </rect>
                </property>
                <property name="currentIndex">
                    <number>0</number>
                </property>
                <widget class="QWidget" name="tabTimer">
                    <attribute name="title">
                        <string>Timer</string>
                    </attribute>
                    <widget class="QTableWidget" name="taskList">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>161</y>
                                <width>561</width>
                                <height>191</height>
                            </rect>
                        </property>
                        <property name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="showDropIndicator" stdset="0">
                            <bool>false</bool>
                        </property>
                        <property name="dragDropOverwriteMode">
                            <bool>false</bool>
                        </property>
                        <property name="alternatingRowColors">
                            <bool>true</bool>
                        </property>
                        <property name="selectionBehavior">
                            <enum>QAbstractItemView::SelectRows</enum>
                        </property>
                        <property name="showGrid">
                            <bool>true</bool>
                        </property>
                        <property name="sortingEnabled">
                            <bool>false</bool>
                        </property>
                        <property name="columnCount">
                            <number>7</number>
                        </property>
                        <attribute name="horizontalHeaderDefaultSectionSize">
                            <number>80</number>
                        </attribute>
                        <attribute name="verticalHeaderVisible">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="verticalHeaderHighlightSections">
                            <bool>true</bool>
                        </attribute>
                        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                            <bool>false</bool>
                        </attribute>
                        <attribute name="verticalHeaderStretchLastSection">
                            <bool>false</bool>
                        </attribute>
                        <row>
                            <property name="text">
                                <string>1</string>
                            </property>
                        </row>
                        <column>
                            <property name="text">
                                <string>ID</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Task Name</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Project</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Date</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Start</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>End</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string/>
                            </property>
                        </column>
                    </widget>
                    <widget class="QWidget" name="formLayoutWidget_2">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>10</y>
                                <width>281</width>
                                <height>101</height>
                            </rect>
                        </property>
                        <layout class="QFormLayout" name="taskLayout">
                            <item row="0" column="0">
                                <widget class="QLabel" name="taskNameLabel">
                                    <property name="text">
                                        <string>Task Name</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QLineEdit" name="taskName"/>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="taskStartLabel">
                                    <property name="text">
                                        <string>Start</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QDateTimeEdit" name="taskStart"/>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="taskEndLabel">
                                    <property name="text">
                                        <string>End</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QDateTimeEdit" name="taskEnd"/>
                            </item>
                            <item row="3" column="0">
                                <widget class="QLabel" name="projectInputLabel">
                                    <property name="text">
                                        <string>Project</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="1">
                                <widget class="QComboBox" name="projectCombo"/>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="horizontalLayoutWidget">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>120</y>
                                <width>561</width>
                                <height>31</height>
                            </rect>
                        </property>
                        <layout class="QHBoxLayout" name="taskButtonLayout">
                            <item>
                                <widget class="QPushButton" name="taskSubmit">
                                    <property name="text">
                                        <string>Create Task</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="taskUpdate">
                                    <property name="text">
                                        <string>Update Task</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="verticalLayoutWidget">
                        <property name="geometry">
                            <rect>
                                <x>299</x>
                                <y>9</y>
                                <width>271</width>
                                <height>101</height>
                            </rect>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QPushButton" name="timerButton">
                                    <property name="text">
                                        <string>Start</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTimeEdit" name="timerDisplay">
                                    <property name="displayFormat">
                                        <string>HH:mm:ss</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </widget>
                <widget class="QWidget" name="tabDashboard">
                    <attribute name="title">
                        <string>Dashboard</string>
                    </attribute>
                    <widget class="QTableWidget" name="dashboardWidegt">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>70</y>
                                <width>561</width>
                                <height>281</height>
                            </rect>
                        </property>
                        <column>
                            <property name="text">
                                <string>Project</string>
                            </property>
                        </column>
                        <column>
                            <property name="text">
                                <string>Duration</string>
                            </property>
                        </column>
                    </widget>
                    <widget class="QWidget" name="formLayoutWidget_3">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>10</y>
                                <width>561</width>
                                <height>51</height>
                            </rect>
                        </property>
                        <layout class="QFormLayout" name="filterLayour">
                            <item row="0" column="0">
                                <widget class="QLabel" name="dffLabel">
                                    <property name="text">
                                        <string>From</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QDateEdit" name="dateFromFilter"/>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="dtfLabel">
                                    <property name="text">
                                        <string>To</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QDateEdit" name="dateToFilter"/>
                            </item>
                        </layout>
                    </widget>
                </widget>
                <widget class="QWidget" name="tabProjects">
                    <attribute name="title">
                        <string>Projects</string>
                    </attribute>
                    <widget class="QWidget" name="formLayoutWidget">
                        <property name="geometry">
                            <rect>
                                <x>9</x>
                                <y>9</y>
                                <width>571</width>
                                <height>128</height>
                            </rect>
                        </property>
                        <layout class="QFormLayout" name="formLayout">
                            <item row="0" column="0">
                                <widget class="QLabel" name="projectLabel">
                                    <property name="text">
                                        <string>Project Name</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QLineEdit" name="projectInput"/>
                            </item>
                            <item row="1" column="1">
                                <widget class="QPushButton" name="projectSubmit">
                                    <property name="text">
                                        <string>Create Project</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QPushButton" name="projectUpdate">
                                    <property name="enabled">
                                        <bool>false</bool>
                                    </property>
                                    <property name="text">
                                        <string>Update Project</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="1">
                                <widget class="QPushButton" name="projectRemove">
                                    <property name="enabled">
                                        <bool>false</bool>
                                    </property>
                                    <property name="text">
                                        <string>Remove Project</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QListWidget" name="projectList">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>150</y>
                                <width>571</width>
                                <height>201</height>
                            </rect>
                        </property>
                    </widget>
                </widget>
            </widget>
        </widget>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources/>
    <connections/>
</ui>
